sdk/eventhub/Azure.Messaging.EventHubs/samples/Sample2_ClientWithCustomOptions.cs

```diff
@@ -68,7 +68,7 @@ await using (var client = new EventHubClient(connectionString, eventHubName, cli
 
                 Console.WriteLine("The Event Hub has the following properties:");
                 Console.WriteLine($"\tThe path to the Event Hub from the namespace is: { properties.Path }");
-                Console.WriteLine($"\tThe Event Hub was created at: { properties.CreatedAtUtc.ToString("yyyy-MM-dd hh:mm:ss tt") }, in UTC.");
```

Comment:
Let's please keep the ", in UTC" text. That's still relevant to those looking at the data; while the structure is self-descriptive, that doesn't necessarily translate to what a user is seeing.

Reply:
Done.